<launch>
  <arg name="fixed_frame" default="openni_depth_frame" />
  
  <node pkg="skeleton_markers" name="skeleton_markers" type="markers_from_skeleton_msg.py" output="screen">
    <rosparam file="$(find fight_game_robot)/params/marker_params.yaml" command="load" />
    <param name="fixed_frame" value="$(arg fixed_frame)" />
  </node>
</launch>
